content/docs/fundamentals/converting/resolving-errors@id.md

```diff
@@ -0,0 +1,350 @@
+
+```html
+<style amp-custom>
+
+/* Konten dari base.css */
+
+</style>
+```
+
+Mari kita selesaikan error-nya:
+
+1.  **Hapus** tag `<link>` yang mengarah ke stylesheet di `<head>` dan ganti dengan tag `<style amp-custom></style>` inline. Atribut `amp-custom` wajib disertakan di tag gaya.
+2. **Salin** semua gaya dari file [`base.css`](https://github.com/googlecodelabs/accelerated-mobile-pages-foundations/blob/master/base.css) ke tag `<style amp-custom></style>`.
+
+Sekali lagi, **muat ulang** halaman dan verifikasi bahwa error stylesheet sudah tidak muncul.
+
+{% call callout('Catatan', type='note') %}
+Tidak hanya gaya inline yang diperlukan tetapi ada batas ukuran file sebesar 50 kilobyte untuk semua informasi gaya. Anda harus menggunakan prapemrosesan CSS seperti [SASS](http://sass-lang.com/) untuk memperkecil CSS sebelum membuat CSS di halaman AMP menjadi inline.
+{% endcall %}
+
+{% call callout('Penting', type='caution') %}
+Anda hanya dapat memiliki 1 tag gaya di seluruh dokumen AMP. Jika Anda memiliki beberapa stylesheet eksternal yang dirujuk oleh halaman AMP, Anda harus menyusun stylesheet ini menjadi 1 kumpulan aturan. Untuk mempelajari tentang aturan CSS apa yang berlaku di AMP, baca [CSS yang Didukung](/id/docs/design/responsive/style_pages.html).
+{% endcall %}
+
+## Jangan sertakan JavaScript pihak ketiga
+
+Meskipun stylesheet dapat dikerjakan ulang dengan relatif mudah menggunakan AMP dengan membuat CSS menjadi inline, hal yang sama tidak berlaku untuk JavaScript.
+
+<pre class="error-text">
+Tag 'script' tidak diizinkan, kecuali dalam bentuk yang spesifik.
+</pre>
+
+Secara umum, skrip di AMP hanya diizinkan jika mengikuti 2 persyaratan utama:
+
+1.  Semua JavaScript harus asinkron (yaitu, menyertakan atribut `async` di tag skrip).
+2.  JavaScript ditujukan untuk library AMP dan komponen AMP apa pun di halaman.
+
+Ini secara efektif mengesampingkan penggunaan semua JavaScript buatan pengguna/pihak ketiga dalam AMP, kecuali seperti disebutkan di bawah.
+
+{% call callout('Catatan', type='note') %}
+Pembatasan pada skrip buatan pengguna/pihak ketiga tidak berlaku untuk skrip berikut:
+
+1.  Skrip yang menambahkan metadata ke halaman atau yang mengonfigurasi komponen AMP. Skrip ini akan memiliki atribut jenis `application/ld+json` atau `application/json`.
+2.  Skrip yang disertakan dalam iframe.  Menyertakan JavaScript dalam iframe harus dianggap sebagai upaya terakhir. Jika memungkinkan, fungsi JavaScript harus diganti menggunakan [Komponen AMP](/id/docs/reference/components.html). Kita akan mempelajari komponen AMP pertama di bagian selanjutnya.
+{% endcall %}
+
+Coba buka file [`base.js`](https://github.com/googlecodelabs/accelerated-mobile-pages-foundations/blob/master/base.js) eksternal. Apa yang Anda lihat? File tidak boleh berisi kode JavaScript sama sekali dan hanya boleh menyertakan komentar informasi sebagaimana berikut:
+
+```javascript
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).

Reply:
Updated code block with EN version.